core/templates/dev/head/pages/topic_editor/TopicEditorNavbarDirective.js

```diff
@@ -23,17 +23,17 @@ oppia.directive('topicEditorNavbar', [
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/pages/topic_editor/topic_editor_navbar_directive.html'),
       controller: [
-        '$scope', '$uibModal', 'AlertsService',
+        '$scope', '$uibModal', '$rootScope', 'AlertsService',
```

Comment:
Why add $rootScope? Don't see it being used...

Reply:
Done.